azext_iot/_help.py

```diff
@@ -820,6 +820,31 @@
         --metric-type system
 """
 
+helps[
+    "iot hub configuration test-queries"
+] = """
+    type: command
+    short-summary: Validates target condition and custom metric queries for a configuration on the IoT Hub
+    long-summary: |
+      This command supports both inline json and file input for custom metric queries `--custom-metric-queries`.
+      Review examples and parameter description for details on how to fully utilize the operation.
+
+      - For bash inline json format use '{"key":"value"}' and \\ (backslash) for command continuation.
+      - For powershell inline json format use '{\\"key\\":\\"value\\"}' and ` (tilde) for command continuation.
+      - For cmd inline json format use \"{\\"key\\":\\"value\\"}\" and ^ (caret) for command continuation.
+      - For file based json input use "@/path/to/file". File based input avoids shell quotation issues.
+
+    examples:
+    - name: Validate the target condition and custom metric queries with inline json.
+      text: >
+        az iot hub configuration test-queries -n {iothub_name} --target-condition "from devices.modules where tags.building=9"
+        --custom-metric-queries '{\\"mymetric\\":""select moduleId from devices.modules where tags.location=''US''""}'
+    - name: Validate the target condition and custom metric queries with json file.
+      text: >
+        az iot hub configuration test-queries -n {iothub_name} --target-condition "from devices.modules where tags.building=9"
+        --custom-metric-queries '/path/to/file'
```

Comment:
Does the file have only one query?

Reply:
nope, you can have multiple queries, the input is a dic[str, str]

Reply:
how does it work if there are multiple queries?

lets say that the first query passes, second fails, third passes - what would the result be?
what if all queries fail?

Reply:
^ on this note, add an example with multiple inline queries

Reply:
updated with multiple inline queries. the return type of query error is dict[str, str], therefore when we have multiple queries with mixed fail/pass, it will have error under the specific queries name

Reply:
here's and example:

(env) PS D:\azure-iot-cli-extension> az iot hub configuration test-queries -n dfsdf --custom-metric-queries '{\"mymetric\":\"sd\", \"mymetric2\":\"select moduleId from devices.modules where tags.location=''US''\"}'
[
  "Custom metric query validation failed: {'mymetric': \"SQL query parse failed due to following errors: \\n\\tLine 1:0 missing SELECT at 'sd'\"}."
]